modules/build/src/main/scala/scala/build/Build.scala

```diff
@@ -256,9 +256,34 @@ object Build {
     val (testBuild, extraTestBuilds) =
       value(buildScope(Scope.Test, Some(mainBuild), Some(extraBuilds)))
 
+    copyResourceToClassesDir(mainBuild)
+    copyResourceToClassesDir(testBuild)
+
     Builds(Seq(mainBuild, testBuild), Seq(extraBuilds, extraTestBuilds))
   }
 
+  private def copyResourceToClassesDir(build: Build) = build match {
+    case b: Build.Successful =>
+      for {
+        resourceDirPath  <- b.sources.resourceDirs
+        resourceFilePath <- os.walk(resourceDirPath)
+        if os.isFile(resourceFilePath)
+        relativeResourcePath = resourceFilePath.relativeTo(resourceDirPath)
+        if !relativeResourcePath.toString.startsWith(
+          ".scala"
+        ) // dismiss files generated by scala-cli
```

Comment:
I want to avoid included files generated by scala-cli to jars. For example, if I run `scala-cli package Hello.scala`
```
head Hello.scala
// using resourceDir "."
```
all `.scala` files would appear in the jar in this case

Reply:
I think they wouldn't even with my suggested changes (which make it ignore the `.scala` directory right under `resourceDirPath`).